pkg/virtctl/ssh/ssh.go

```diff
@@ -76,9 +76,9 @@ func AddCommandlineArgs(flagset *pflag.FlagSet, opts *SSHOptions) {
 	flagset.IntVarP(&opts.SSHPort, portFlag, portFlagShort, opts.SSHPort,
 		fmt.Sprintf(`--%s=22: Specify a port on the VM to send SSH traffic to`, portFlag))
 	flagset.StringArrayVarP(&opts.AdditionalSSHLocalOptions, additionalOpts, additionalOptsShort, opts.AdditionalSSHLocalOptions,
-		fmt.Sprintf(`--%s="-o StrictHostKeyChecking=no" : Additional options to be passed to the local ssh. This is applied only if local-ssh=true `, commandToExecute))
-	flagset.BoolVar(&opts.WrapLocalSSH, wrapLocalSSHFlag, opts.WrapLocalSSH,
-		fmt.Sprintf("--%s=true: Set this to true to use the SSH/SCP client available on your system by using this command as ProxyCommand; If set to false, this will establish a SSH/SCP connection with limited capabilities provided by this client", wrapLocalSSHFlag))
+		fmt.Sprintf(`--%s="-o StrictHostKeyChecking=no" : Additional options to be passed to the local ssh. This is applied only when using the local ssh client.`, commandToExecute))
```

Comment:
now that you make this a compile time decision, mentioning a difference between local ssh client and built-in one does not seem to make any sense ...

Reply:
You could move also this option in the function `addAdditionalCommandlineArgs`

Reply:
btw. independent of that, I would recommend introducing a "normal" flag for disabling host key checking. Supplying this seems super cumbersome for users ...